packages/widgetbook/lib/src/app/router.dart

```diff
@@ -24,51 +24,32 @@ GoRouter createRouter({
 }) {
   return GoRouter(
     initialLocation: initialLocation,
-    redirect: (context, state) {
-      // Redirect deprecated `disable-navigation` and `disable-properties`
-      // query params to their equivalent `panels` query param.
-      if (state.queryParams.containsKey('disable-navigation') ||
-          state.queryParams.containsKey('disable-properties')) {
-        final disableNavigation = _parseBoolQueryParameter(
-          value: state.queryParams['disable-navigation'],
-        );
-
-        final disableProperties = _parseBoolQueryParameter(
-          value: state.queryParams['disable-properties'],
-        );
-
-        final panels = {
-          if (!disableNavigation) ...{
-            WidgetbookPanel.navigation,
-          },
-          if (!disableProperties) ...{
-            WidgetbookPanel.addons,
-            WidgetbookPanel.knobs,
-          }
-        };
-
-        return '/?panels={${panels.map((x) => x.name).join(",")}}';
-      }
-
-      return null;
-    },
```

Comment:
The cloud needs to support both, the preview as well as the old approach but I think it should be fine

Reply:
By the old approach, do you mean the `panels` or the `disable-*`? Because the latter is supported.

Reply:
I mean the `disable-*`

Reply:
They are supported in the code you commented on below.